app/main.py

```diff
@@ -105,6 +106,7 @@ async def home(request: Request, db: Session = Depends(get_db)):
     quote = daily_quotes.quote_per_day(db)
     return templates.TemplateResponse("index.html", {
         "request": request,
+        "message": "Hello, World!",
```

Comment:
Please remove.

Reply:
i dont create this function.you need to talk with the one that create the quote tiket(mabey its neccary to her)

Reply:
You restored this line, so you did add it. It was removed a while back.

Reply:
ok i will delete it